app/src/test/java/com/example/whatshouldiread/customclasses/QuestionTest.java

```diff
@@ -21,4 +21,25 @@ public void subtractionWorks()
     {
         assertEquals(2, 4-2);
     }
-}
\ No newline at end of file
+
+    @Test
```

Comment:
For both the book and question tests, we're missing coverage for the entirety of the class. @moburke Do you mind adding in the missing coverage?

Reply:
@moburke I created an example test that uses the given, when, then format to test the default constructor for the Question Class. Use that as an example for the book class and to finish the question class.
Commit: 2088c815431abf737433ddb4c82ccba0410d6288